### DISCUSSION TITLE
Can't access my Birdnet-Pi and no detections anymore since trying new model

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#728] [Q&A]
Hello,

Three days ago I switched to the new model and it worked fine for one day, with lot of detections. At the end of the first day I decided to change a few settings to fine tune detections and the following day, I only had two detections so I decided to go back with the old model, time to read more about the new model.

Unfortunately, today, day 3, I can't access my settings on the web page of my Birdnet-Pi, they don't work anymore, username/password seem to be wrong (I didn't change them). I can still ssh into my Birdnet-Pi.

And I have 0 detections for day 3.

Thank you for any suggestions.

Edit:

Here is a screen capture of my view log, it seems there is a problem with my geographical coordinates: 

![eQ3tZYs](https://user-images.githubusercontent.com/83965802/217105710-54057e36-9c3a-4a38-b936-416eb0f85fa6.png)

**Comment by ehpersonal38** [comment]
According to the screenshot I'd say this is an issue with your birdnet.conf - can you check that file and tell me what you find? The location is `~/BirdNET-Pi/birdnet.conf`

**Reply by blogdemoi** [reply]
> APPRISE_NOTIFICATION_TITLE="New BirdNET-Pi Detection"
> APPRISE_NOTIFICATION_BODY="A $sciname $comname was just detected with a confide>
> APPRISE_NOTIFY_EACH_DETECTION=0
> APPRISE_NOTIFY_NEW_SPECIES=0
> APPRISE_NOTIFY_NEW_SPECIES_EACH_DAY=0
> APPRISE_MINIMUM_SECONDS_BETWEEN_NOTIFICATIONS_PER_SPECIES=0
> DATABASE_LANG=not-selected
> RTSP_STREAM=
> FLICKR_API_KEY=
> FLICKR_FILTER_EMAIL=
> APPRISE_WEEKLY_REPORT=1
> SILENCE_UPDATE_INDICATOR=0
> FREQSHIFT_TOOL=sox
> FREQSHIFT_HI=6000
> FREQSHIFT_LO=3000
> FREQSHIFT_PITCH=-1500
> HEARTBEAT_URL=
> MODEL=BirdNET_6K_GLOBAL_MODEL
> SF_THRESH=0.03

Thank you.

**Reply by blogdemoi** [reply]
There should be a Flickr api key...

**Reply by ehpersonal38** [reply]
Is that the entire file? If so then it looks like a lot of lines got deleted.

The following link is a blank template of birdnet.conf. I recommend renaming your existing `birdnet.conf` to `birdnet.conf.old` and then copy and pasting the entire contents of the template into a new file `~/BirdNET-Pi/birdnet.conf`). Then fill out your latitude and longitude to 4 decimal places (example `38.4937`), and then try accessing your settings again. The username will be `birdnet` and the password will be blank.

https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/birdnet.conf-defaults

If this works then you can refer back to `birdnet.conf.old` to get your Flickr API keys and other information to put it in your new config.

**Reply by blogdemoi** [reply]
> Is that the entire file? If so then it looks like a lot of lines got deleted.

Yes, it is the entire file and I didn't do anything except switching back from new model to old model. I will follow your recommendations and report back. :-)

**Reply by blogdemoi** [reply]
I copied/pasted the content to ~/BirdNET-Pi/birdnet.conf, added my coordinates, could access my web settings, filled in my Flickr ID and other infos in the web page, saved them successfully.

But when I tried to access the advaced settings it asked again for username/password, tried to fill in again with birnetpi/blank psswd but it didn't accept them and couldn't access the adavanced settings page.

I tried the above procedure again (I copied/pasted the content to ~/BirdNET-Pi/birdnet.conf, added my coordinates, could access my web settings), but this time after being able to access the basic settings page and having reloaded the web page I have a blank page with nothing.

I can still ssh though. I rebooted too but still a blank page.

**Reply by ehpersonal38** [reply]
@blogdemoi Well dang... I don't really know what to tell you. Is it possible for you to reinstall? Backup your `birds.db` with this command: `cp ~/BirdNET-Pi/scripts/birds.db ~/birds.db`

Then after reinstall you can replace it:
`cp ~/birds.db ~/BirdNET-Pi/scripts`

**Comment by blogdemoi** [comment]
Thank you for your help. I will reinstall. I reinstalled it before, flashing again the Raspberry Pi on the SD card then installing Birdnet-Pi on it.

I am wondering if I can simply leave the SD card in my Pi as it is now, uninstall Bitdnet-pi then reinstall Bitdnet-Pi. (as I can still SSH into it)?

Or should I start from scratch flashing the SD card then install Birdnet-Pi?

**Reply by ehpersonal38** [reply]
> I am wondering if I can simply leave the SD card in my Pi as it is now, uninstall Bitdnet-pi then reinstall Bitdnet-Pi. (as I can still SSH into it)?

I would try this option first!

**Reply by blogdemoi** [reply]
I just uninstalled then reinstalled as I was still connected to the Pi via ssh. Everything went fine, I imported my backed-up birds.db  and I chose the new model. Waiting for the birds now.

Thank you for your help.
